Discovery/Discovery/Presentation/WebDiscovery/DiscoverWebviewModel.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  DiscoverWebviewModel.swift
+//  Discovery
+//
+//  Created by SaeedBashir on 12/16/23.
+//
+
+import Foundation
+import Core
+import WebKit
+import SwiftUI
+import Swinject
```

Comment:
Can we remove this import? Looks like we're not using it.

Reply:
Done